### PR TITLE
Fix tslint errors

### PR DESCRIPTION
**Pull request by fabien0102**
Some tslint was in this file. I've just fixed these errors and improve a little bit type :angel:

Normally, Travis should passed properly after this PR

### CODE REVIEW

File: packages/visual-tests/src/TestCases/Chart/symbol-3.ts [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
+}
+
+type Country = keyof typeof generators
+
+const dataPoint = (country: Country) => {
   return generators[country]()
 }
 
-const generateData = (country, amount = 100) => {
-  let result = []
```
Comment (fabien0102) [inline packages/visual-tests/src/TestCases/Chart/symbol-3.ts]: Even if I understand the intention, tslint was complaining about this line. Indeed `result` is never reassigned so it should be a `const` :wink: (`Array.push` is just a method that mutate the array)

**Comment by fabien0102** [comment]
Pfff… out of memory randomly on Travis… 😕

File: packages/visual-tests/src/TestCases/Chart/symbol-3.ts (hunk 1 above)
Comment (micha-f) [reply packages/visual-tests/src/TestCases/Chart/symbol-3.ts]: looks good to me @fabien0102

**Comment by fabien0102** [comment]
Part of #484 (rebased to be able to have success tests)

**Pull request closed without merging**